playground/eShopLite/AppHost/AppHost.csproj

```diff
@@ -10,9 +10,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting\Aspire.Hosting.csproj" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Azure\Aspire.Hosting.Azure.csproj" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Azure.Provisioning\Aspire.Hosting.Azure.Provisioning.csproj" />
+    <ProjectReference Include="..\..\..\src\Aspire.Hosting\Aspire.Hosting.csproj" AppHostLibrary="true" />
```

Comment:
So this will be required anytime you want to add a reference to a library and not treat it like a resource project?

Reply:
Correct.

@rainersigwald suggested that we could add a build warning if we detect a `<ProjectReference` to a `Library` project and tell you that it won't get referenced by default, suggesting to add `AppHostLibrary="true"` to the ProjectReference. I think that is a good idea and am going to implement it here.

BTW - anyone have a better name than `AppHostLibrary` for this property?

Reply:
How solid is the term `AppHost`? It feels pretty broad - my gut is that the property should be scoped with `Aspire` in some way to narrow it somewhat. Maybe `AspireHostLibrary`? Total gut feel though.

Reply:
> @rainersigwald suggested that we could add a build warning if we detect a <ProjectReference to a Library project and tell you that it won't get referenced by default, suggesting to add AppHostLibrary="true" to the ProjectReference. I think that is a good idea and am going to implement it here.

I have added this build warning. I needed to write a C# task, but we don't have a task library today to put it in. So I used a RoslynCodeTaskFactory to get unblocked here. Once this is merged I plan on logging an issue to move this code into a proper Tasks .dll. I'm trying to not get blocked by adding that infrastructure here.